### PR TITLE
Xcode 12.0 GM build settings

### PR DESCRIPTION
**Pull request by segiddins**
Will require code changes, but this shows the default build settings diff

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* xcode 12 beta 1 build settings  
  [segiddins](https://github.com/segiddins)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Review by dnkoutso — APPROVED** [review]

File: lib/xcodeproj/constants.rb [hunk 1]
```diff
@@ -381,6 +380,7 @@ module Constants
         'CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF'    => 'YES',
         'CLANG_WARN_OBJC_LITERAL_CONVERSION'      => 'YES',
         'CLANG_WARN_OBJC_ROOT_CLASS'              => 'YES_ERROR',
+        'CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER' => 'YES',
```
Comment (segiddins) [inline lib/xcodeproj/constants.rb:383]: I think this might require an update in cocoapods to turn it off?

Comment (paulb777) [reply lib/xcodeproj/constants.rb:383]: Should already be done in #9905

Comment (amorde) [reply lib/xcodeproj/constants.rb:383]: Yup already done in https://github.com/CocoaPods/CocoaPods/pull/9905

Comment (amorde) [reply lib/xcodeproj/constants.rb:383]: sorry I hadn't refreshed before commenting :D

**Pull request merged**